build.gradle

```diff
@@ -562,6 +562,7 @@ project(':clients') {
     compile libs.lz4
     compile libs.snappy
     compile libs.slf4jApi
+    compile libs.jacksonDatabind
```

Comment:
Wait, no, I feel like we're not on the same page. I don't think we want to add the dependency to clients. That would mean we'd eventually have to move it to a separate jar, which would make it source incompatible between Kafka releases. I was suggesting we should just figure out a reasonable organization of jars or use some fine-grained jars because at least those you can collapse into a single jar and maintain compatibility for awhile using empty jars that just depend on the collapsed one. I don't know how you make compatibility and migration work if you dump everything into kafka-clients.